packages/jbrowse-web/src/plugins/HierarchicalTrackSelectorDrawerWidget/components/Category.js

```diff
@@ -7,6 +7,7 @@ import Typography from '@material-ui/core/Typography'
 import { observer, PropTypes as MobxPropTypes } from 'mobx-react'
 import propTypes from 'prop-types'
 import React from 'react'
+// eslint-disable-next-line import/no-cycle
```

Comment:
@rbuels Is there a reason you didn't disable the import/no-cycle with the other eslint fixes? I think it should be fine, but let me know if there's something about the component structure you think needs to be fixed.

Reply:
Was just to prompt you to have a look at those cycles when you got around to it. :-)